src/components/index-button.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Chevron from "react-chevron";
+import { Toolbar, ToolbarItem, useToolbarState } from "reakit";
+import { useAppDispatch, useAppSelector } from "../app/hooks";
+import { navActions, nextPrevParentSelector } from "../features/nav/navSlice";
+
+export function IndexButton() {
+    const toolbar = useToolbarState();
+    const navTargets = useAppSelector(nextPrevParentSelector);
+    const dispatch = useAppDispatch();
+
+    return (
+            <ToolbarItem
+                {...toolbar}
+                as="a"
+                href="index-1.html"
```

Comment:
I don't like hardcoded URLs...Is the index in the manifest? This should conditionally render depending on whether an index exists, right?